doc/manual/src/command-ref/nix-shell.md

```diff
@@ -88,7 +88,9 @@ All options not listed here are passed to `nix-store
   cleared before the interactive shell is started, so you get an
   environment that more closely corresponds to the “real” Nix build. A
   few variables, in particular `HOME`, `USER` and `DISPLAY`, are
-  retained.
+  retained.  Note that the shell used to run commands is obtained from
+  `NIX_BUILD_SHELL` / `<nixpkgs>` from `NIX_PATH`, and therefore not
```

Comment:
Apparently [the note on `NIX_BUILD_SHELL`](https://github.com/NixOS/nix/pull/8809/files#diff-3eb54def6708c86fc4695323b8a91cd918c8e7d63b80d14d26a0385952c6176bL119) is wrong. This is what actually happens:

https://github.com/NixOS/nix/blob/621c23bbea9db9138d64a8111f6183672d85c5af/src/nix-build/nix-build.cc#L432-L443

Reply:
I went over the code and the only discrepancy I've noticed is the package used -  `bash` vs. `bashInteractive`, is there anything else I missed?

Reply:
It seems I had some blooper there. Not sure what my problem was... Looks fine now.